eng/packages/http-client-csharp/generator/Azure.Generator/src/Primitives/KnownAzureTypes.cs

```diff
@@ -50,6 +59,7 @@ private static ValueExpression DeserializeParsableStringLikeType(CSharpType valu
             [ETagId] = typeof(ETag),
             [AzureLocationId] = typeof(AzureLocation),
             [ArmIdId] = typeof(ResourceIdentifier),
+            [AzureError] = typeof(ResponseError),
```

Comment:
Should this be handled in https://github.com/Azure/azure-sdk-for-net/pull/49653?

Reply:
No - there isn't a need to model the ResponseError as a TypeProvider. The SystemObjectTypeProvider is specific to the case where you need to model as a TypeProvider because the type belongs in the type hierarchy of other type providers.